.bashrc

```diff
@@ -0,0 +1,4 @@
+# Ruby exports
```

Comment:
@pranayag15 why did you create this file?

Reply:
We don't need this file please delete this.

Reply:
I don't know how it got installed but I will change it back to what it was.